cheljoh/purrrsonality_test
# Refactoring party

@rrgayhart @brianrip

What's this PR do?
- Broke functions into different files. Refactored the portion of the app that renders personality results card (was crazy long html before).

Where should the reviewer start?
- purrrsonality_test.js has the original code. The file was eventually deleted. 

How should this be manually tested?
- go to localhost:3000 and take the test. Make sure the results are rendered correctly and there are no obvious bugs. 

Any background context you want to provide?
- Initially difficult to refactor because of the styling. Eventually had to make new divs and append the cards there. 

Screenshots (if appropriate)

What gif best describes this PR or how it makes you feel? 

<iframe src="//giphy.com/embed/Ti3i7TumKRkdi" width="480" height="264" frameBorder="0" class="giphy-embed" allowFullScreen></iframe><p><a href="http://giphy.com/gifs/crazy-laughing-Ti3i7TumKRkdi">Gizmo</a></p>
